mobify/platform-scaffold
# WEBDATA-101 Retain email address in checkout flow

When checking out on Merlin's the confirmation email is not filled in again when going back to the Shipping step. Also, the country field is being reset to `US` during the initialization of the shipping page.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-101

## Issue

![6oyfi4myn2](https://cloud.githubusercontent.com/assets/10948652/26808533/4f76a7fc-4a12-11e7-81a1-1933df2a0853.gif)

## Changes
- Update checkout redux form initial value, username(email) is included now
- Update the email field onBlur handler, to avoid sending request with empty email provided
- Added `isInitialized` flag for shipping page

## How to test-drive this PR
- (necessary config changes)
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add your favourite potion to the cart
- Fill out shipping information
- Continue to payment
- Go back to shipping page and verify the form fields
